test/ProgressOnderwijsUtilsTests/XmlCompressionTest.cs

```diff
@@ -149,9 +150,7 @@ public void SaveToUtf8ExcludesIndentation()
             var bytes = XmlCompression.ToUtf8(doc);
             var str = UTF8.GetString(bytes);
 
-            PAssert.That(() => str == @"<test><nested><elements><here>
-                Ƒϕϕ
-            </here></elements></nested></test>");
+            PAssert.That(() => str == $"<test><nested><elements><here>{Environment.NewLine}                Ƒϕϕ{Environment.NewLine}            </here></elements></nested></test>");

```

Comment:
This is odd; git misconfiguration?

Reply:
or XDocument bug?

Reply:
(Want de oorsprong van deze newlines is in deze file, en die zullen dezelfde chars zijn als hier, dus iets zit hier foutief slim te doen.

Reply:
https://stackoverflow.com/questions/2369207/preserve-carriage-return-when-using-xdocument-parse

XDocument.Parse bug.